crm_compassion/models/weekly_demand.py

```diff
@@ -158,7 +158,7 @@ def _default_unsponsored_ambassador(self):
         """
         start_date = datetime.today() - timedelta(weeks=STATS_DURATION)
         ambass_sponsored = self.env['recurring.contract'].search_count([
-            ('origin_id.type', '=', 'partner'),
+            ('parent_id', '!=', False),
```

Comment:
Here the fix is wrong! it's only the type 'sub' that we want to correct, so please remove this change.